ios/GigyaSdkWrapper.swift

```diff
@@ -60,6 +68,10 @@ class GigyaSdkWrapper<T: GigyaAccountProtocol>: GigyaSdkWrapperProtocol {
         self.promise = promise
 
         switch name {
+        case .getSession:
+            _ = self.getSession()
+        case .setSession:
+            self.setSession(token: params["token"] as! String, secret: params["secret"] as! String, expiration: params["expiration"] as! NSNumber)
```

Comment:
Expiration should be double and no NSNumber.

Reply:
Updated.